icu4c/source/i18n/unicode/ucoleitr.h

```diff
@@ -102,6 +102,10 @@ typedef struct UCollationElements UCollationElements;
 /**
  * Open the collation elements for a string.
  *
+ * The UCollationElements retains a pointer to the supplied text.
+ * The caller must not modify or delete the text while the UCollationElements
+ * retains the reference, until after calling <tt>ucol_setText</tt>.
```

Comment:
also until ucol_closeElements()

Maybe make it more generic? "... while the UCollationElements object is used to iterate over this text"?

Similar for the other places.

I prefer `<code>` over `<tt>`.

Reply:
Good idea: If it's more generic text then we don't need to update the comments if more APIs are added/etc.

Re: `<code>` vs `<tt>`: I like the `<code>` too, but used `<tt>` since the file seems to only use that. If you like I'll change the whole file to use `<code>` instead.